invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/deposit/RDMDepositForm.js

```diff
@@ -424,6 +425,12 @@ export class RDMDepositForm extends Component {
                   labelIcon="tag"
                 /> */}
 
+                  <CitationField
+                    label={"Citation"}
+                    fieldPath={"metadata.citation"}
+                    schema="citation"
+                  />
+
```

Comment:
Per https://github.com/inveniosoftware/react-invenio-deposit/pull/299#pullrequestreview-675728912 , the citation is generated on the landing page. It shouldn't be on the deposit page.